src/templates/snek/gql/queries/user.ts

```diff
@@ -0,0 +1,64 @@
+//#region > Imports
+//#PACKAGE "graphql-tag"
+//## npm install "graphql-tag"@2.10.3
+// Contains a gql tag for wrapping queries
+import gql from "graphql-tag";
+//#endregion
+
+//#region > Queries
+/**
+ * Whoami.
+ *
+ * @deprecated since 1.0 use jwtAuth instead
```

Comment:
Is it true tho?

Reply:
Nope. That tag should be removed. `whoami` is not deprecated!